pctiler/Dockerfile.dev

```diff
@@ -1,6 +1,8 @@
 FROM pc-apis-tiler
 
-COPY requirements-dev.txt requirements-dev.txt
-RUN pip install -r requirements-dev.txt
+ARG MAMBA_DOCKERFILE_ACTIVATE=1
 
-RUN pip install -e ./pccommon -e ./pctiler
+COPY --chown=$MAMBA_USER:$MAMBA_USER requirements-dev.txt requirements-dev.txt
+
+RUN python3 -m pip install -r requirements-dev.txt
+RUN python3 -m pip install -e ./pccommon -e ./pctiler
```

Comment:
Just a note that these lines are *slightly* risky, in that something in pip-land could decide to (re-)install that something that we expect to get from conda (like rasterio).

I manually verified that things were working as expected (that pip wasn't installing rasterio) but things might change in the future, and we wouldn't necessarily know until stuff breaks.